src/Service/ListServiceInterface.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace OpenBibIdApi\Service;
+
+interface ListServiceInterface extends ServiceInterface
+{
+
+    /**
+     * Get the lists of the current logged in user.
+     *
+     * @return \DOMDocument|null
+     *   A \DOMDocument containing the XML from the response, null if HTTP
+     *   status code 204 (No content) was returned.
+     *
+     * @throws BibApi\Exception\BibException
```

Comment:
@throws moet enkel toegevoegd worden in de documentatie als de methode zelf een exception throwed. 

Indien een exception getrhowed wordt door het aanroepen van een andere methode of methode van ander object dan dient deze niet in de dochead gedocumenteerd te worden.